### DISCUSSION TITLE
GitHub Actions Just stopped working

### DISCUSSION THREAD

**Discussion by ScentAndCo** [community/community#77212] [Actions]
### Select Topic Area

Question

### Body

Hi

I set up some github actions to automatically push files from my repository to my server when I push to certain branches, here is the yml file I am using to do that: 
on:
  push:
    branches:
      - staging
name: 🚀 Deploy website on push - Staging
jobs:
  web-deploy:
    name: 🎉 Deploy
    runs-on: ubuntu-latest
    steps:
    - name: 🚚 Get latest code
      uses: actions/checkout@v3
    
    - name: 📂 Sync files
      uses: SamKirkland/FTP-Deploy-Action@v4.3.4
      with:
        server: myServerAddress
        username: ${{ secrets.ftp_username }}
        password: ${{ secrets.ftp_password }}
        local-dir: myLocalFilePath
        server-dir: myServerFilePath

up until about an hour ago this was working perfectly and had been for weeks but then it just stopped, whenever I publish a new change to the repo no workflow is triggered so my files are not pushed to my server which is making me less efficient. Does anyone have an idea why this has happened as no error messages or anything are given and how I can go about fixing it, my working guess is that I have hit some sort of usage cap but I haven't been informed of this. Any Help is much appreciated

**Comment by srelf-eschercloud** [comment]
i think its the on_push event thats not being actioned.

**Comment by facundofarias** [comment]
I know you are trying to use a Github Action, but maybe you can give [DeployHQ](https://www.deployhq.com/) a try!
